lib/interactive_viewer.ml

```diff
@@ -167,6 +168,10 @@ let view (patches : Patch.t list) =
                  | `ASCII 'h', [] ->
                      Lwd.set help true;
                      `Handled
+                 | `ASCII 't', [] ->
```

Comment:
Why did you chose `t` ?
This is not consistent with keybindings that users might already know as in `less`, it's used to jump between tags and in [`tig`](https://github.com/jonas/tig/blob/master/doc/manual.adoc#default-keybindings), it's used to toggle the tree view.

Reply:
What about using `m` since we are returning to the top which in some sense is the main view. Or what do you suggest?

Reply:
I would suggest to use the same keybinding as `less`!

Reply:
so a `g` ?